apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/cql/ExpressionUtils.java

```diff
@@ -0,0 +1,33 @@
+package io.stargate.sgv2.api.common.cql;
+
+import com.bpodgursky.jbool_expressions.And;
+import com.bpodgursky.jbool_expressions.Expression;
+import com.bpodgursky.jbool_expressions.Or;
+import java.util.List;
+
+public class ExpressionUtils<K> {
+
+  public static <K> And<K> OrderedAndOf(Expression<K>... expressions) {
```

Comment:
Is there need for non-ordered one? If not, just use `andOf()` and `orOf()`. Can clarify in Javadoc(s) that expressions are constructed in specified order.

Reply:
done